src/tests/parser_test.py

```diff
@@ -888,6 +888,12 @@ def test():
         def test():
             pass
     """),
+    CodeSnippet("""\
+        '''Test this'''
+
+        @property
```

Comment:
Can you also add some tests for multiple decorators applied to single line function?

Reply:
Sure, Added this scenario in the parameterised test.
without adding a new test i can't test that both decorators are recorded, will add it later if you think it's needed.